src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -54,7 +54,7 @@ def test_self_cal(session):
 
 def test_configure_arb_waveform(session):
     waveform_data = [0.000000, 0.049068, 0.098017, 0.146730, 0.195090, 0.242980, 0.290285, 0.336890, 0.382683, 0.427555]
-    session.output_mode = nifgen.OutputMode.NIFGEN_VAL_OUTPUT_ARB  # TODO(Jaleel): name to change per #553
+    session.output_mode = nifgen.OutputMode.ARB  # TODO(Jaleel): name to change per #553
```

Comment:
You can remove the TODO since you are fixing it and closing #553

Reply:
Removed